modules/@ergonode/attributes/src/extends/components/Forms/AttributeFormParamsSelect.vue

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <Select
+        :data-cy="dataCyGenerator('params')"
+        key="attrHasParams"
+        :value="value"
+        required
+        :label="paramsLabel"
+        :options="attributeParametersOptions"
+        :error-messages="errorMessages[paramsFieldKey]"
+        :disabled="!isAllowedToUpdate"
+        @input="setParameterValue" />
+</template>
+
+<script>
+import PRIVILEGES from '@Attributes/config/privileges';
+import {
+    toCapitalize,
+} from '@Core/models/stringWrapper';
+import {
+    mapActions,
+    mapState,
+} from 'vuex';
+
+export default {
+    name: 'AttributeFormParamsSelect',
+    components: {
+        Select: () => import('@Core/components/Inputs/Select/Select'),
+    },
+    props: {
+        errorMessages: {
+            type: Object,
+            default: () => ({}),
+        },
+        typeKey: {
+            type: String,
+            default: '',
+        },
+        getParams: {
```

Comment:
Passing function is not good idea from Vue perspective. Could we do it in other way?

Reply:
not at the moment